src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -1992,7 +1992,7 @@ dumpBabelPhysicalDatabaseACLs(Archive *fout)
 					"\n	SET LOCAL ROLE sysadmin;"
 					"\n	FOR rolname, original_name IN ("
 					"\n		SELECT a.rolname, a.orig_username FROM sys.babelfish_authid_user_ext a"
-					"\n			WHERE orig_username IN ('dbo','db_accessadmin') AND"
+					"\n			WHERE orig_username IN ('dbo','db_accessadmin','db_securityadmin') AND"
```

Comment:
redeisgn/rethink this query so that we dont need to re-write this everytime

Reply:
This needs to be backward compatible, so this will be an interesting challenge

Reply:
Need @rishabhtanwar29 's opinion on this.